src/utils/index.js

```diff
@@ -0,0 +1 @@
+export { ActionCreator } from "./actions";
```

Comment:
You've deleted the ActionCreator type I think, so this is no longer valid

Reply:
Ah yes, great point, will change in a new PR